main.go

```diff
@@ -102,3 +132,26 @@ func prettyPrint(expectedPrs []PullRequestDetails, fileName string) error {
 	err = t.Execute(f, expectedPrs)
 	return err
 }
+
+func saveIntoFileReleases(releaseList []ReleaseDetails, fileName string) error {
```

Comment:
[nitpick] Could reuse the file save method, because it is doing json marshalling in both cases.

Reply:
Added `report-utils.go` file, with the report generation related functions.